config/locales/en.yml

```diff
@@ -80,27 +49,33 @@ en:
     mark_as_finished: Mark as finished
     mark_as_inactive: Mark as inactive
     my_gardens: My Gardens
-    new_seeds: New saved seed
+    new_harvest: New harvest
     new_planting: New planting
     new_post: Write new post
-    new_harvest: New harvest
+    new_seeds: New saved seed
     plant: Plant
     plant_crop: Plant %{crop_name}
+    plant_something: Plant something
     plant_something_here: Plant something here
     record_harvest: Record Harvest
     save_seeds: Save seeds
     timeline: Timeline
     write_blog_post: Write blog post
   crops:
-    search: Search crops
     index:
       subtitle: "%{crops_size} total"
       title: Browse Crops
     requested:
       link: You have %{number_crops} crops awaiting approval
       subtitle: Pending approval
       title: Requested crops
+    search: Search crops
+  date:
+    abbr_month_names: Abbr month names
```

Comment:
https://guides.rubyonrails.org/i18n.html#adding-date-time-formats - this should be a rails built in.